optimism/src/keccak/constraints.rs

```diff
@@ -210,9 +210,8 @@ impl<Fp: Field> Constraints for KeccakEnv<Fp> {
                     let rot_e = Self::from_quarters(&self.vec_dense_rot_e(), Some(y), x);
 
                     self.constrain(
-                        self.is_round()
```

Comment:
You're removing `self.is_round()` here.

Reply:
WHY? Thanks for pointing out, no idea how that one got in, probably a merge issue because I actually added this in an old PR.

Reply:
Look, it was added here https://github.com/o1-labs/proof-systems/pull/1823/files

Reply:
I think I will merge master in this branch before proceeding just in case other changes are being reverted unintendedly